backend/src/org/commcare/util/CommCarePlatform.java

```diff
@@ -32,25 +30,23 @@
 public class CommCarePlatform implements CommCareInstance {
     // TODO: We should make this unique using the parser to invalidate this ID or something
     public static final String APP_PROFILE_RESOURCE_ID = "commcare-application-profile";
-
-    private Vector<Integer> suites;
```

Comment:
Pretty strange that  this isn't used... The ODK level `AndroidCommCarePlatform` class has an equivalent that is used, but I'm guess the J2ME code somehow doesn't use this...